modules/monitoring-support-considerations.adoc

```diff
@@ -28,6 +28,8 @@ This procedure is a supported exception to the preceding statement.
 * *Modifying resources of the stack.* The {product-title} monitoring stack ensures its resources are always in the state it expects them to be. If they are modified, the stack will reset them.
 * *Deploying user-defined workloads to `openshift-&#42;`, and `kube-&#42;` projects.* These projects are reserved for Red Hat provided components and they should not be used for user-defined workloads.
 * *Enabling symptom based monitoring by using the `Probe` custom resource definition (CRD) in Prometheus Operator.*
+* *Manually deploying monitoring resources into namespaces with the `openshift.io/cluster-monitoring: "true"` label.* 
```

Comment:
Is the "with" here maybe more accurately described as "that have," "by using," or something else? Not immediately clear to me as I parachute in.

Reply:
Thank you for pointing out this possible double meaning, it means "that have". I will change it so that it should not cause confusion, thank you!